File: app/src/main/java/org/dhis2/usescases/login/LoginComponent.java
package org.dhis2.usescases.login;


import android.support.annotation.NonNull;

import org.dhis2.data.dagger.PerActivity;
import org.dhis2.usescases.synchronization.SynchronizationComponent;
import org.dhis2.usescases.synchronization.SynchronizationModule;

import dagger.Subcomponent;

@PerActivity
@Subcomponent(modules = LoginModule.class)
public interface LoginComponent {
    void inject(LoginActivity loginActivity);
}

File: app/src/main/java/org/dhis2/usescases/login/LoginComponent.kt
package org.dhis2.usescases.login

import dagger.Subcomponent
import org.dhis2.commons.di.dagger.PerActivity
import org.dhis2.data.biometric.BiometricModule

@PerActivity
@Subcomponent(modules = [LoginModule::class, BiometricModule::class])
interface LoginComponent {
    fun inject(loginActivity: LoginActivity)
}
